Package@swift-5.9.swift

```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:5.9
```

Comment:
You don't need a separate Package.swift for 5.9 you can set 5.9 in the main Package.swift
It will compile in swift 6 mode because you have added the `.version("6")`

Reply:
Understand. Thanks. 9cb04eb